frontend/src/main/pages/StaffEditPage.js

```diff
@@ -0,0 +1,13 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+export default function StaffEditPage() {
+
+  // Stryker disable all : placeholder for future implementation
+  return (
+    <BasicLayout>
+      <div className="pt-2">
+        <h1>Edit page not yet implemented</h1>
+      </div>
+    </BasicLayout>
+  )
+}
```

Comment:
Not implemented, because there's nothing to implement.   There's literally no fields to edit.

Please remove edit from the PR so that it is mergeable.